MANIFEST.in

```diff
@@ -2,7 +2,7 @@ recursive-include setuptools *.py *.exe *.xml *.tmpl
 recursive-include tests *.py
 recursive-include setuptools/tests *.html
 recursive-include docs *.py *.txt *.rst *.conf *.css *.css_t Makefile indexsidebar.html
-recursive-include setuptools/_vendor *.py *.txt
+recursive-include setuptools/_vendor *
```

Comment:
Add a `recursive-exclude setuptools/_vendor tests`?

Reply:
Tell me more. Why not leave the vendored packages installed the same way that they are installed in other environments? If they're present in the source tree, they should probably be present in the sdist.

I did a quick check and the only `tests` is `_vendor/importlib_resources/tests`. I'd really like to avoid taking responsibility to tweak the vendored packages in any way.

Reply:
The tests here are failing because there is an explicit test that no tests are included in the vendored packages.

It was just a proposal to fix the failing test here.

Reply:
Since I'm disinclined to follow the suggestion and because it's unrelated to this change (those tests directories would have been included by way of `*.py`, I'm going to dismiss the suggestion for now. If you feel strongly about it or especially if you have a use-case that's impacted by the lack of this setting, feel free to file a separate bug to track. Thanks.

Reply:
I don't understand: I saw this PR with the exact changes that are here now fail with a CI failure due to the included test packages... anyways, it was only a suggestion to make that test pass, which seems to not fail anymore now, so all good.

I don't really understand what happend

Reply:
Aah yes. Sorry the test suite is so confusing. Setuptools [enables `-rx`](https://github.com/pypa/setuptools/blob/6d915ca1b67d43609714a70bff23526a362dd0f1/pytest.ini#L7) (report xfailures), so when any test fails, the xfails are also reported. It makes it really difficult to discern what the real issues are.

The reason the tests were failing is because there was a lint error (an extra newline) failing the lint checks. I fixed that by force-pushing https://github.com/pypa/setuptools/compare/b9d9b2999419e3e0bb59dd5f1baa55bb3e5a917a..65e00b6e96c531c2b0be023475bb956ebc976c39.

Thanks for explaining your thinking and motivating me to file a new issue to just remove the noisy reporting.